assignments/callbacks.js

```diff
@@ -27,19 +27,32 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
```

Comment:
function getLength(arr, cb) {
  // getLength passes the length of the array into the callback.
  console.log(cb(arr));
}

let arrayLength = function(items) {
  return items.length;
}

getLength(items,arrayLength);